README.md

```diff
@@ -98,7 +98,7 @@ need to relink it.
 
 ## Should I Use It?
 
-Yes! The web-ext tool enables you to build and ship web extensions for Firefox.
+Yes! The web-ext tool enables you to build and ship WebExtensions for Firefox.
```

Comment:
I think this is a good opportunity to generalize this statement to: "...enables you to build and ship extensions for Firefox..."